integration_test/third_party_apps_test/applications/iisv1/metadata.yaml

```diff
@@ -0,0 +1,138 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+public_url: "https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/third-party/iis"
+app_url: "https://docs.microsoft.com/iis/"
+short_name: IIS
+long_name: Internet Information Services
+logo_path: /stackdriver/images/iis.png # supplied by google technical writer
+description: |-
+  The Internet Information Services (IIS) integration collects telemetry from
+  your IIS web servers. The metrics provide connection information
+  and also data on transferred bytes. The integration also collects information
+  from access logs.
+
+  When using the Ops Agent on a Microsoft Windows VM, the agent automatically
+  collects IIS metrics. No additional configuration is required.
+minimum_supported_agent_version:
+  metrics: 1.0.0
+  logging: 2.14.0
+supported_operating_systems: windows
+supported_app_version: ["8.5", "10.0"]
+expected_metrics:
+  # v1 metrics
```

Comment:
Question: Are the v1 "expected_metrics" used for anything in the integration tests? (We don't use these to build the tables of agent metrics that appear in the docs (unless those metrics are derived from this using a mechanism I don't know about). So I was just curious why this info is maintained here. So this is purely a curiosity question.

Reply:
Yes, the tests actually check that the metrics are present and conform to the expected format. In fact, I just had to exclude non-sqlserver images from testing because the tests could not find the relevant metrics there.

Reply:
Thanks! Like I said, just curious!